### PR TITLE
Include a constant for DCPower type ID

### PR DESCRIPTION
**Pull request by dixonjoel**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds a Type ID constant for the 'niDCPower' string. Add this to the DCPower palette and use it in the code modules. We can use this PR to set a pattern and a follow up PR will add constants for the remaining 4 drivers.
image
image

### Why should this Pull Request be merged?

Use this constant in multiple places instead of hard-coding the string all over.
Start of [AB#2552370](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2552370)

### What testing has been done?

Installed the .vip built locally and ensure the DC Power example isn't broken.

### CODE REVIEW

File: Source/Runtime/Measurement Server.lvproj [hunk 1]
```diff
@@ -132,6 +132,7 @@
 			<Item Name="niFgen_64.dll" Type="Document" URL="niFgen_64.dll">
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
+			<Item Name="Initialize and Close Session Behavior.ctl" Type="VI" URL="../MeasurementLink Session Management/Session Reservation/Initialize and Close Session Behavior.ctl"/>
```
Comment (pbirkhol-ni) [inline Source/Runtime/Measurement Server.lvproj:135]: Strange. This is in Jonathan's PR, which hasn't even been submitted yet.

File: Source/Runtime/MeasurementLink Session Management/Session Reservation/Create Session Reservation.vi
Comment (pbirkhol-ni) [inline Source/Runtime/MeasurementLink Session Management/Session Reservation/Create Session Reservation.vi:1]: Why did this VI change? I think you can revert it.

**Review by pbirkhol-ni — APPROVED** [review]

File: Source/Runtime/MeasurementLink niDCPower/Instrument Type ID.vi
[the remark this answers is not in the document]
Comment (jasonmreding) [reply Source/Runtime/MeasurementLink niDCPower/Instrument Type ID.vi:1]: - Consider making this an inline VI
- Consider making this part of the class library instead.
- I don't know what sort of iconography is in flight for this. Lacking anything official, my recommendation would be to try and make it look more like the string constants on the string palette. That means pink outline and not header/banner. Not sure what would go in the middle, but ideally height would be the same.

[the remark this answers is not in the document]
Comment (dixonjoel) [reply Source/Runtime/MeasurementLink niDCPower/Instrument Type ID.vi:1]: > Consider making this part of the class library instead.

You mean a, e.g. public VI under the class, right?

> I don't know what sort of iconography is in flight for this. Lacking anything official, my recommendation would be to try and make it look more like the string constants on the string palette. That means pink outline and not header/banner. Not sure what would go in the middle, but ideally height would be the same.

For now, I think I'll try to make it look like a string constant and put the driver name inside like it has in the header now? We wanted some homage to the driver. Maybe we can have a pink outline, but a driver-like fill color? Anyway, I'll play around with it and like paul said, we should get icons made for us anyway.

File: Source/Runtime/Measurement Server.lvproj (hunk 1 above)
Comment (dixonjoel) [reply Source/Runtime/Measurement Server.lvproj:135]: Weird. I think I'll clean things out and kind of start over on this.

**Pull request merged**